cluster-autoscaler/FAQ.md

```diff
@@ -31,21 +31,21 @@ this document:
   * [How can I prevent Cluster Autoscaler from scaling down a particular node?](#how-can-i-prevent-cluster-autoscaler-from-scaling-down-a-particular-node)
 * [Internals](#internals)
   * [Are all of the mentioned heuristics and timings final?](#are-all-of-the-mentioned-heuristics-and-timings-final)
-  * [How does scale up work?](#how-does-scale-up-work)
+  * [How does scale-up work?](#how-does-scale-up-work)
   * [How does scale down work?](#how-does-scale-down-work)
```

Comment:
if we have scale-up then we should also have scale-down

Reply:
Agreed, will fix. Thanks!